tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
+    "module": "commonjs",
+    "target": "es5",
+    "sourceMap": true,
+    "moduleResolution": "node",
+    "rootDirs": ["src"],
+    "jsx": "react",
+    "removeComments": true,
+    "forceConsistentCasingInFileNames": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
Kill it with :fire:
Seriously this allow stuff like :
```
const foo: { stuff: string } = {stuff: ''};
const data = 'notstuff';
const bar = foo[data];
```
This option is old, now we can use `keyof` for better typings:
```
const foo: { stuff: string } = {stuff: ''};
const data: keyof { stuff: string } = 'stuff';
const bar = foo[data];
```